bot/src/main/java/org/opendaylight/bot/Client.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2018 Red Hat, Inc. and others. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v1.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.opendaylight.bot.Jenkins;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Map;
+import java.io.IOException;
+
+import com.offbytwo.jenkins.JenkinsServer;
+import com.offbytwo.jenkins.model.JobWithDetails;
+import com.offbytwo.jenkins.model.Job;
+
+import org.opendaylight.bot.BotConfiguration;
+
+
+/**
+ * Template of Initial Jenkins Client API.
+ *
+ * @author Prateek Chanda
+ */
+@SuppressFBWarnings({"NP_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD", "UWF_UNWRITTEN_FIELD", "UWF_UNWRITTEN_PUBLIC_OR_PROTECTED_FIELD"})
+
+public class Client {
+
+    String admin;
+    String password;
+    String jobName;
+    public BotConfiguration botConfiguration;
+
+    void getAdmin(String newAdmin) {
```

Comment:
sorry but these methods make no sense to me.. you probably meant for this to be set* not get* ? But if you pass BotConfiguration to the constructor then these setters are not required...

Reply:
I just wanted to make this more modular.

I guess setting all the botConfiguration variables within a single function will do good